### ISSUE TITLE
Disabling archive functionality

### ISSUE THREAD

**Issue by ab623** [sissbruecker/linkding#258]
I don't intend to utilise the archiving feature of this application. I intend to hide the archive behind a use settings.

Tasks required
- Hide archive button on bookmarks page
- Hide archive button on main nav
- Create user setting to hide/show archive
- Update docs. 

@sissbruecker is this a feature you would merge if I raised a PR for it?

**Comment by sissbruecker** [comment]
Sounds interesting, and shouldn't add too much complexity. You would also have to provide tests. In this case we want to ensure that the setting is disabled by default, that archive buttons and links are visible by default, and that they are hidden when the feature is enabled. Here is a PR with a similar scope: https://github.com/sissbruecker/linkding/pull/164/files

**Comment by ab623** [comment]
Good point on the tests. Will work on this when I have some free time and raise a PR.

**Comment by schaafjs** [comment]
Something else I'd add for completeness which probably requires testing: if a user has archived entries and tries to disable the archive, they should be notified somehow.
Either allow them to abort (and nothing changes) or continue (and all archived entries are unarchived).

**Comment by ab623** [comment]
Yes, but also a nice to have. Its a non-destructive change, all it is doing is hiding the UI elements. So if there is an issue it can be reverted back.

Likely, on the settings page, I can add some commentary along the lines of "Enabling this feature will hide any existing archived linked from view"

**Comment by schaafjs** [comment]
Yeah, that also works. It would probably be more fitting to call it "Hide archive" or similar then since nothing actually gets disabled.
